haiku-apps/xrs/xrs-1.9.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A simple rhythm station for creating music loops"
+DESCRIPTION="XRS is a simple Rhythm Station. You can use it to create music compositions and loops of your preferred elettronic music genres."
+HOMEPAGE="https://github.com/Freaxed/xrs-haiku"
+COPYRIGHT="2006-2022, Andrea Anzani."
```

Comment:
Looking into the source I also found Stephan Aßmus name, maybe you could add him too?
```
Copyright (C) 2001-2008 Stephan Aßmus. All rights reserved.
```

Reply:
Sure! will double check all the files to double check any missing..